src/pages/Routes.tsx

```diff
@@ -1,29 +1,73 @@
 import { NavBar } from '../components/NavigationBar';
-import { Box, Button, Text} from 'native-base';
-import { useNavigate, createSearchParams } from 'react-router-dom';
-
+import { Box, Divider, Flex, Text } from 'native-base';
+import { useState, useEffect } from 'react';
+import { Route } from '../xplat/types/route';
+import { getActiveRoutesCursor, getArchivedRoutesCursor } from '../xplat/api';
+import { RouteRow } from '../components/RouteRow';
 
 const Routes = () => {
-    const navigate = useNavigate();
-    const exampleSearchParams = {uid: 'GQBdclAMmE2v4nDPphsc'};
-    const navToRoute = () => {
-        navigate({
-            pathname: '/route',
-            search: `?${createSearchParams(exampleSearchParams)}`
-        });
-    };
+    const [activeRoutes, setActiveRoutes] = useState<Route[]>([]);
+    const [archivedRoutes, setArchivedRoutes] = useState<Route[]>([]);
+
+    useEffect(() => {
+        const fetchActiveRoutes = async () => {
+            const activeRoutesCursor = getActiveRoutesCursor();
+            const tempActiveRoutes: Route[] = [];
+            while ((await activeRoutesCursor.hasNext())) {
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                tempActiveRoutes.push((await activeRoutesCursor.pollNext())!);
+            }
+            setActiveRoutes(tempActiveRoutes);
+        };
+        fetchActiveRoutes();
+    }, []);
+
+    useEffect(() => {
+        const fetchArchivedRoutes = async () => {
+            const archivedRoutesCursor = getArchivedRoutesCursor();
+            const tempArchivedRoutes: Route[] = [];
+            while ((await archivedRoutesCursor.hasNext())) {
+                // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
+                tempArchivedRoutes.push((await archivedRoutesCursor.pollNext())!);
+            }
+            setArchivedRoutes(tempArchivedRoutes);
+        };
+        fetchArchivedRoutes();
+    }, []);
 
     return (
         <Box flexDir={'column'}>
-            <Box height={'50px'} marginBottom={1}><NavBar/></Box>
-            <Box>
-                <Button onPress={navToRoute}>
-                    <Text variant={'button'}> Go to Route</Text>
-                </Button>
-            </Box>
+            <Box height={'50px'} marginBottom={1}><NavBar /></Box>
+            <Flex flexDirection='row' justifyContent='space-evenly' width='100%' top='50px'>
+                <Flex flexDirection='row' justifyContent='center' width='30%'>
+                    <Flex flexDirection='column' alignItems='center' width='100%'>
+                        <Text>Active Routes</Text>
+                        {
+                            activeRoutes.map((currRoute: Route, index: number) => (
+                                <>
```

Comment:
I would like for us to implement clickable RouteRow components so that the employees can click on a route from this screen and navigate to the route details or route feed page directly. I can implement this today if I can find the time.

Reply:
Nevermind brother, just saw that you already implemented this. This looks good to me, we can update the look of the lists later